girder_worker/plugins/docker/tests/docker_integration_test.py

```diff
@@ -0,0 +1,278 @@
+import ConfigParser
+import girder_worker
+import os
+import shutil
+import six
+import stat
+import sys
+import unittest
+
+from girder_worker.core import run, io
+
+test_image = 'cjh1/girder_worker:test'
```

Comment:
@zachmullen Can you create me ( or give me permission to create ) a girder_worker repo.